src/block/BrewingStand.php

```diff
@@ -145,4 +145,8 @@ public function onScheduledUpdate() : void{
 			}
 		}
 	}
+
+	public function getDropsForIncompatibleTool(Item $item) : array{
```

Comment:
Can be replaced by ``getDrops`` because it will drop exactly the same no matter how it's broken